src/codeu/chat/common/ServerInfo.java

```diff
@@ -1,14 +1,17 @@
 package codeu.chat.common;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+
 import codeu.chat.util.Uuid;
 import codeu.chat.util.Time;
 
 public final class ServerInfo {
   private final Time startTime;
   private final Uuid version;
   private final static String SERVER_VERSION = "1.0.0";
-  
+
```

Comment:
nit: you should generally avoid adding new-lines and whitespaces here and there. e.g there are 4 of those in this file. make your change as concise as possible. makes for easier reviewing too